eng/scripts/CodeCheck.ps1

```diff
@@ -47,7 +47,12 @@ function LogError {
 try {
     if ($ci) {
         # Install dotnet.exe
-        & $repoRoot/restore.cmd -ci -NoBuildNodeJS -DotNetRuntimeSourceFeed $DotNetRuntimeSourceFeed -DotNetRuntimeSourceFeedKey $DotNetRuntimeSourceFeedKey
+        if ($DotNetRuntimeSourceFeed -or $DotNetRuntimeSourceFeedKey) {
```

Comment:
@mmitche I think this was failing since the variables DotNetRuntimeSourceFeed  and DotNetRuntimeSourceFeedKey may be empty and will fail when passed to build.ps1. Does this sound right to you?

Reply:
We should be able to accept both cases.